src/Hilke.KineticConvolution/Helpers/ShapeBuilder.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Fractions;
+
+using MoreLinq;
+
+namespace Hilke.KineticConvolution.Helpers
+{
+    internal class ShapeBuilder<TAlgebraicNumber>
+    {
+        private readonly ConvolutionFactory<TAlgebraicNumber> _convolutionFactory;
+        private readonly IAlgebraicNumberCalculator<TAlgebraicNumber> _calculator;
+        private readonly GeometryCalculator<TAlgebraicNumber> _geometryCalculator;
+        private Point<TAlgebraicNumber>? _startPoint;
+        private readonly List<(Point<TAlgebraicNumber> point, TAlgebraicNumber radius)> _corners;
```

Comment:
move it above `_startPoint` with the other readonly fields.

Reply:
done